src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/srm/algebras/BV64Algebra.cs

```diff
@@ -131,15 +136,19 @@ public BDD ConvertToCharSet(ICharAlgebra<BDD> solver, ulong pred)
             if (_partition == null)
                 throw new NotImplementedException(nameof(ConvertToCharSet));
 
+            // the result will be the union of all minterms in the set
```

Comment:
We probably need a glossary somewhere in the code that describes what various terms mean in the context used in the source: minterms, algebra, etc.

Reply:
Would likely be helpful. For minterms the MintermGenerator class' XML comment also includes a short description/example, which other XML comments could link to with `<see cref="MintermGenerator{TPredicate}"/>`. Visual Studio at least seems to make that a clickable link.

Reply:
I can help Olli with some of these missing comments.